2024-03-06 17:17:36,191 INFO    MainThread:71408 [internal.py:wandb_internal():90] W&B internal server running at pid: 71408, started at: 2024-03-06 17:17:36.191971
2024-03-06 17:17:36,193 INFO    WriterThread:71408 [datastore.py:open_for_write():75] open: .\wandb\run-20240306_171734-1wq3lei6\run-1wq3lei6.wandb
2024-03-06 17:17:36,196 DEBUG   SenderThread:71408 [sender.py:send():235] send: header
2024-03-06 17:17:36,197 DEBUG   SenderThread:71408 [sender.py:send():235] send: run
2024-03-06 17:17:36,199 INFO    SenderThread:71408 [sender.py:_maybe_setup_resume():492] checking resume status for None/UNet-DAS/1wq3lei6
2024-03-06 17:17:39,051 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: check_version
2024-03-06 17:17:39,059 INFO    SenderThread:71408 [dir_watcher.py:__init__():205] watching files in: .\wandb\run-20240306_171734-1wq3lei6\files
2024-03-06 17:17:39,060 INFO    SenderThread:71408 [sender.py:_start_run_threads():812] run started: 1wq3lei6 with start time 1709716655
2024-03-06 17:17:39,060 DEBUG   SenderThread:71408 [sender.py:send():235] send: summary
2024-03-06 17:17:39,061 INFO    SenderThread:71408 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2024-03-06 17:17:39,061 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: check_version
2024-03-06 17:17:39,910 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: run_start
2024-03-06 17:17:39,947 DEBUG   HandlerThread:71408 [meta.py:__init__():35] meta init
2024-03-06 17:17:39,947 DEBUG   HandlerThread:71408 [meta.py:__init__():49] meta init done
2024-03-06 17:17:39,948 DEBUG   HandlerThread:71408 [meta.py:probe():209] probe
2024-03-06 17:17:39,970 DEBUG   HandlerThread:71408 [meta.py:_setup_git():199] setup git
2024-03-06 17:17:40,062 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_created():256] file/dir created: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-summary.json
2024-03-06 17:17:40,118 DEBUG   HandlerThread:71408 [meta.py:_setup_git():206] setup git done
2024-03-06 17:17:40,118 DEBUG   HandlerThread:71408 [meta.py:_save_pip():53] save pip
2024-03-06 17:17:40,119 DEBUG   HandlerThread:71408 [meta.py:_save_pip():67] save pip done
2024-03-06 17:17:40,119 DEBUG   HandlerThread:71408 [meta.py:_save_conda():74] save conda
2024-03-06 17:17:41,069 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_created():256] file/dir created: .\wandb\run-20240306_171734-1wq3lei6\files\requirements.txt
2024-03-06 17:17:41,070 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_created():256] file/dir created: .\wandb\run-20240306_171734-1wq3lei6\files\conda-environment.yaml
2024-03-06 17:17:43,087 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\conda-environment.yaml
2024-03-06 17:17:43,159 DEBUG   HandlerThread:71408 [meta.py:_save_conda():84] save conda done
2024-03-06 17:17:43,159 DEBUG   HandlerThread:71408 [meta.py:probe():247] probe done
2024-03-06 17:17:43,234 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: stop_status
2024-03-06 17:17:43,234 DEBUG   SenderThread:71408 [sender.py:send():235] send: files
2024-03-06 17:17:43,234 INFO    SenderThread:71408 [sender.py:_save_file():949] saving file wandb-metadata.json with policy now
2024-03-06 17:17:43,236 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: stop_status
2024-03-06 17:17:44,098 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_created():256] file/dir created: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:17:44,098 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_created():256] file/dir created: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-metadata.json
2024-03-06 17:17:45,063 DEBUG   SenderThread:71408 [sender.py:send():235] send: telemetry
2024-03-06 17:17:45,063 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:17:45,063 DEBUG   SenderThread:71408 [sender.py:send():235] send: telemetry
2024-03-06 17:17:45,063 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:17:45,063 WARNING SenderThread:71408 [sender.py:send_metric():905] Seen metric with glob (shouldn't happen)
2024-03-06 17:17:45,063 DEBUG   SenderThread:71408 [sender.py:send():235] send: config
2024-03-06 17:17:45,162 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-metadata.json?Expires=1709803065&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=akyQ9fASzldBLyDQc9WyfjSOhcIcxkceIfnl7T2tzk7fLJ1Gjiv6MHoLM81Hd6OagfTR%2FRvTkmGvPdguPkTtHZ%2BFcP7QfiVh1qIfNzH1f0vxDw%2FzqcPGfe9qOBxktDL%2FIBAoFGEkR0LZjvETEUGXGrU0nMt3OuJDhAHJeye2zrU%2FDfwWJs5FwupkVsbfyEvJby5uXld31%2FOO06c65uNApcbw9VA0rPRAlGTvarQNbrFXQ5I38ZDg9Ds2jKlXvfblN%2FWgE04QwFhXjaQ7IBGEP0Dupvh%2FmB1Pd6cEh2EfptzeC3ClurZYbf%2BIdG%2FVAMphRp5lYAkQMl02fi0MEL4uJg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:17:45,162 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '967'}
2024-03-06 17:17:45,162 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:17:46,118 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:17:46,844 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-metadata.json?Expires=1709803065&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=akyQ9fASzldBLyDQc9WyfjSOhcIcxkceIfnl7T2tzk7fLJ1Gjiv6MHoLM81Hd6OagfTR%2FRvTkmGvPdguPkTtHZ%2BFcP7QfiVh1qIfNzH1f0vxDw%2FzqcPGfe9qOBxktDL%2FIBAoFGEkR0LZjvETEUGXGrU0nMt3OuJDhAHJeye2zrU%2FDfwWJs5FwupkVsbfyEvJby5uXld31%2FOO06c65uNApcbw9VA0rPRAlGTvarQNbrFXQ5I38ZDg9Ds2jKlXvfblN%2FWgE04QwFhXjaQ7IBGEP0Dupvh%2FmB1Pd6cEh2EfptzeC3ClurZYbf%2BIdG%2FVAMphRp5lYAkQMl02fi0MEL4uJg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:17:46,844 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '967'}
2024-03-06 17:17:46,844 ERROR   Thread-17 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:17:50,180 INFO    Thread-17 :71408 [upload_job.py:push():132] Uploaded file C:\Users\19086\AppData\Local\Temp\tmp1bm0rf7rwandb\2crw29dn-wandb-metadata.json
2024-03-06 17:18:00,073 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: stop_status
2024-03-06 17:18:00,073 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: stop_status
2024-03-06 17:18:03,278 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:18:05,306 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:18:06,191 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: partial_history
2024-03-06 17:18:06,192 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: metric
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: history
2024-03-06 17:18:06,193 DEBUG   SenderThread:71408 [sender.py:send():235] send: summary
2024-03-06 17:18:06,194 INFO    SenderThread:71408 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2024-03-06 17:18:06,314 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-summary.json
2024-03-06 17:18:07,324 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:18:07,522 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:08,127 DEBUG   SenderThread:71408 [sender.py:send():235] send: telemetry
2024-03-06 17:18:08,127 DEBUG   SenderThread:71408 [sender.py:send():235] send: exit
2024-03-06 17:18:08,128 INFO    SenderThread:71408 [sender.py:send_exit():371] handling exit code: 0
2024-03-06 17:18:08,128 INFO    SenderThread:71408 [sender.py:send_exit():373] handling runtime: 27
2024-03-06 17:18:08,129 INFO    SenderThread:71408 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2024-03-06 17:18:08,129 INFO    SenderThread:71408 [sender.py:send_exit():379] send defer
2024-03-06 17:18:08,129 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:08,130 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,130 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 0
2024-03-06 17:18:08,130 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,131 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 0
2024-03-06 17:18:08,131 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 1
2024-03-06 17:18:08,131 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,131 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 1
2024-03-06 17:18:08,234 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,234 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 1
2024-03-06 17:18:08,234 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 2
2024-03-06 17:18:08,234 DEBUG   SenderThread:71408 [sender.py:send():235] send: stats
2024-03-06 17:18:08,235 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,235 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 2
2024-03-06 17:18:08,235 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,235 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 2
2024-03-06 17:18:08,235 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 3
2024-03-06 17:18:08,235 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,235 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 3
2024-03-06 17:18:08,235 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,236 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 3
2024-03-06 17:18:08,236 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 4
2024-03-06 17:18:08,236 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,236 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 4
2024-03-06 17:18:08,236 DEBUG   SenderThread:71408 [sender.py:send():235] send: summary
2024-03-06 17:18:08,237 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:08,237 INFO    SenderThread:71408 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2024-03-06 17:18:08,238 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,238 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 4
2024-03-06 17:18:08,238 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 5
2024-03-06 17:18:08,238 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:08,238 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:08,238 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 5
2024-03-06 17:18:08,238 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:08,239 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 5
2024-03-06 17:18:08,330 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\config.yaml
2024-03-06 17:18:08,330 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\output.log
2024-03-06 17:18:08,331 INFO    Thread-11 :71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-summary.json
2024-03-06 17:18:08,344 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,323 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 6
2024-03-06 17:18:11,323 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,324 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:11,324 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 6
2024-03-06 17:18:11,324 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:11,324 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 6
2024-03-06 17:18:11,324 INFO    SenderThread:71408 [dir_watcher.py:finish():336] shutting down directory watcher
2024-03-06 17:18:11,341 INFO    SenderThread:71408 [dir_watcher.py:_on_file_modified():269] file/dir modified: .\wandb\run-20240306_171734-1wq3lei6\files\config.yaml
2024-03-06 17:18:11,342 INFO    SenderThread:71408 [dir_watcher.py:finish():366] scan: .\wandb\run-20240306_171734-1wq3lei6\files
2024-03-06 17:18:11,342 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\conda-environment.yaml conda-environment.yaml
2024-03-06 17:18:11,343 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\config.yaml config.yaml
2024-03-06 17:18:11,346 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\output.log output.log
2024-03-06 17:18:11,349 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\requirements.txt requirements.txt
2024-03-06 17:18:11,352 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-metadata.json wandb-metadata.json
2024-03-06 17:18:11,352 INFO    SenderThread:71408 [dir_watcher.py:finish():380] scan save: .\wandb\run-20240306_171734-1wq3lei6\files\wandb-summary.json wandb-summary.json
2024-03-06 17:18:11,355 INFO    SenderThread:71408 [sender.py:transition_state():392] send defer: 7
2024-03-06 17:18:11,355 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: defer
2024-03-06 17:18:11,357 INFO    HandlerThread:71408 [handler.py:handle_request_defer():164] handle defer: 7
2024-03-06 17:18:11,358 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: defer
2024-03-06 17:18:11,358 INFO    SenderThread:71408 [sender.py:send_request_defer():388] handle sender defer: 7
2024-03-06 17:18:11,358 INFO    SenderThread:71408 [file_pusher.py:finish():145] shutting down file pusher
2024-03-06 17:18:11,430 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,430 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,538 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,538 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,645 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,645 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,752 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,752 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,858 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,858 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:11,965 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:11,965 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,071 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,071 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,178 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,178 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,285 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,285 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,391 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,391 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,501 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,501 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,606 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,606 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,713 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,713 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,820 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,820 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:12,929 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:12,929 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,038 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,039 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,143 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,143 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,249 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,249 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,301 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:13,301 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:13,301 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:13,355 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,355 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,363 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:13,363 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:13,363 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:13,382 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/config.yaml?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=nj8ISUnCsU0bwALaSaBdPAOygosGAI7EISZP05accsVNeBesmvsBk5cwTEFD4kLppqclxm8xNsYa1RUjPsPMnHQJsWEdLhnK5N%2BAQbyHFnDZEcc5T%2B3VwPyU9CqlxD%2BPXccFJ8j98n%2FRxXS%2FVCMtnnJj3x3pEBethVkXbnWwCUnfwWj8LNLrm2YucFvFzlcmwVYLouicgadr%2F3hLfL0sHzH%2FIN8JO66tEPSEogj8cWvkLdmGOtoHWsD0f%2BApGaME12i%2Fvs8VkUjc3mxSRO6a%2FdHx4Qgwfc2KUKEhu8lJGeDfyoO9BWrNpcfx1Mqnt%2BTtcP%2BogS0bK%2F8rriu1H3sGmA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:13,382 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1891'}
2024-03-06 17:18:13,382 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:13,420 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/conda-environment.yaml?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=wqoIhmCkkNdSWj2TvoE554QpqJXw%2FRL%2B8zN6kVIiEDEwF7zwzJWqskczQcdcYU%2Fy0tnUThTN%2FNjOi4LqEovBTsVgG30P1m7USC%2B5V1WppCEyj7YU8sEftBPiOa3JCzZE2AFsHiIKBhwvXuf75YGZ%2B6sWQap8out2SdO4aq43KH23010TyeNAj04bf0uu0F80F%2F6Ns9SyssVOVSp8adGoh83cCC7BSgGGQF6NVgv8Z948W23HrRzC%2FbVkWC%2FhqxF5rGHtTLehotTJlKCWCHFbl3tGiZ1Z6Rk%2FUV8rc0%2FIHLPRJ70nJBBW3LSd5uWKSDWLIbqJ80rV%2BChboaJYns3yvw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:13,420 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1975'}
2024-03-06 17:18:13,420 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:13,463 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,463 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,465 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:13,465 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:13,465 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:13,569 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,569 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,676 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,676 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,782 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,783 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,890 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,890 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:13,998 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:13,998 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,107 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,107 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,213 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,213 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,321 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,321 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,429 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,430 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,535 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,535 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,641 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,641 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,748 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,748 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,853 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,853 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:14,959 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:14,959 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,055 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:15,055 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:15,055 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:15,066 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,066 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,083 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:15,083 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:15,083 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:15,138 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/config.yaml?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=nj8ISUnCsU0bwALaSaBdPAOygosGAI7EISZP05accsVNeBesmvsBk5cwTEFD4kLppqclxm8xNsYa1RUjPsPMnHQJsWEdLhnK5N%2BAQbyHFnDZEcc5T%2B3VwPyU9CqlxD%2BPXccFJ8j98n%2FRxXS%2FVCMtnnJj3x3pEBethVkXbnWwCUnfwWj8LNLrm2YucFvFzlcmwVYLouicgadr%2F3hLfL0sHzH%2FIN8JO66tEPSEogj8cWvkLdmGOtoHWsD0f%2BApGaME12i%2Fvs8VkUjc3mxSRO6a%2FdHx4Qgwfc2KUKEhu8lJGeDfyoO9BWrNpcfx1Mqnt%2BTtcP%2BogS0bK%2F8rriu1H3sGmA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:15,138 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1891'}
2024-03-06 17:18:15,138 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:15,172 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,172 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,195 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:15,195 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:15,195 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:15,244 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/conda-environment.yaml?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=wqoIhmCkkNdSWj2TvoE554QpqJXw%2FRL%2B8zN6kVIiEDEwF7zwzJWqskczQcdcYU%2Fy0tnUThTN%2FNjOi4LqEovBTsVgG30P1m7USC%2B5V1WppCEyj7YU8sEftBPiOa3JCzZE2AFsHiIKBhwvXuf75YGZ%2B6sWQap8out2SdO4aq43KH23010TyeNAj04bf0uu0F80F%2F6Ns9SyssVOVSp8adGoh83cCC7BSgGGQF6NVgv8Z948W23HrRzC%2FbVkWC%2FhqxF5rGHtTLehotTJlKCWCHFbl3tGiZ1Z6Rk%2FUV8rc0%2FIHLPRJ70nJBBW3LSd5uWKSDWLIbqJ80rV%2BChboaJYns3yvw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:15,244 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1975'}
2024-03-06 17:18:15,244 ERROR   Thread-18 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:15,279 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,279 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,386 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,386 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,494 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,494 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,602 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,602 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,708 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,708 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,815 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,815 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:15,922 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:15,922 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,028 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,028 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,135 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,135 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,242 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,242 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,347 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,347 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,453 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,453 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,560 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,560 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,666 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,666 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,773 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,773 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,880 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,881 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:16,988 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:16,988 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,094 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,094 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,201 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,201 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,308 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,308 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,415 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,415 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,522 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,522 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,629 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,629 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,706 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:17,707 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:17,707 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:17,735 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,735 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,760 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:17,760 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:17,760 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:17,843 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,843 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:17,862 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/config.yaml?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=nj8ISUnCsU0bwALaSaBdPAOygosGAI7EISZP05accsVNeBesmvsBk5cwTEFD4kLppqclxm8xNsYa1RUjPsPMnHQJsWEdLhnK5N%2BAQbyHFnDZEcc5T%2B3VwPyU9CqlxD%2BPXccFJ8j98n%2FRxXS%2FVCMtnnJj3x3pEBethVkXbnWwCUnfwWj8LNLrm2YucFvFzlcmwVYLouicgadr%2F3hLfL0sHzH%2FIN8JO66tEPSEogj8cWvkLdmGOtoHWsD0f%2BApGaME12i%2Fvs8VkUjc3mxSRO6a%2FdHx4Qgwfc2KUKEhu8lJGeDfyoO9BWrNpcfx1Mqnt%2BTtcP%2BogS0bK%2F8rriu1H3sGmA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:17,862 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1891'}
2024-03-06 17:18:17,863 ERROR   Thread-19 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:17,868 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:17,868 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:17,868 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:17,948 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:17,948 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:18,053 DEBUG   HandlerThread:71408 [handler.py:handle_request():141] handle_request: poll_exit
2024-03-06 17:18:18,053 DEBUG   SenderThread:71408 [sender.py:send_request():249] send_request: poll_exit
2024-03-06 17:18:18,138 WARNING MainThread:71408 [internal.py:wandb_internal():153] Internal process interrupt: 1
2024-03-06 17:18:18,632 INFO    Thread-18 :71408 [upload_job.py:push():132] Uploaded file .\wandb\run-20240306_171734-1wq3lei6\files\conda-environment.yaml
2024-03-06 17:18:22,556 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:22,556 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:22,556 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:23,249 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:23,249 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:23,249 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:23,258 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:23,258 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:23,258 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:23,919 INFO    Thread-19 :71408 [upload_job.py:push():132] Uploaded file .\wandb\run-20240306_171734-1wq3lei6\files\config.yaml
2024-03-06 17:18:26,203 WARNING MainThread:71408 [internal.py:is_dead():387] Internal process exiting, parent pid 68024 disappeared
2024-03-06 17:18:26,203 ERROR   MainThread:71408 [internal.py:wandb_internal():149] Internal process shutdown.
2024-03-06 17:18:26,350 INFO    WriterThread:71408 [datastore.py:close():279] close: .\wandb\run-20240306_171734-1wq3lei6\run-1wq3lei6.wandb
2024-03-06 17:18:27,112 INFO    SenderThread:71408 [sender.py:finish():1111] shutting down sender
2024-03-06 17:18:27,112 INFO    HandlerThread:71408 [handler.py:finish():806] shutting down handler
2024-03-06 17:18:27,112 INFO    SenderThread:71408 [file_pusher.py:finish():145] shutting down file pusher
2024-03-06 17:18:27,112 INFO    SenderThread:71408 [file_pusher.py:join():150] waiting for file pusher
2024-03-06 17:18:32,223 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:32,223 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:32,224 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:32,335 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:32,335 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:32,335 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:32,519 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:32,520 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:32,520 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:49,015 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:49,015 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:18:49,015 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:50,279 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:50,279 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:18:50,279 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:18:51,166 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:18:51,167 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:18:51,167 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:19:26,096 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:19:26,096 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:19:26,096 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:19:26,630 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:19:26,630 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:19:26,630 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:19:28,325 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:19:28,326 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:19:28,326 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:20:34,679 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:20:34,679 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:20:34,679 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:20:34,833 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:20:34,834 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:20:34,834 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:20:38,857 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:20:38,858 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:20:38,858 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:22:49,318 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/wandb-summary.json?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=kQGab5RyQRyNNm%2BkITYfnTTkmglmSpy%2BncSKUvmpFSR6hl%2FNFtRXIWwAQbYUNU7vqjseQ0YILU%2B17BdNxMA3beZ8s8jxaX56ppP6O1yNGKYCKooWpxVVyH8i9Cr%2FV8Igy942f%2B%2FalGsS2AJLmSLoN6QUEfq1%2BHPkcgQ%2BTo%2F5rxu4J2u6gSdXSOn0kg93tNlMekY6HfWNeWu%2B0z6O0iIVu5flU9281wCQ84ZGsspVP%2FM1WqEOFxTe4cf2VsHbHyZk7qflr2z4Yy5GsjgP64BRHJVNbECJpod3CromxsP6SkYf%2FJJNjrGXpDMtw1EObf54cnkehx5ncGWiFa7RakWAMw%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:22:49,318 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '290'}
2024-03-06 17:22:49,318 ERROR   Thread-22 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:23:08,640 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/requirements.txt?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=CEne8lm6YEMuaqVOdEP7HgP2ho8ka5GSHr47z%2BOasnJs6FbR5CXXjSWviuI42v9ovBCcdqhl6iXzy2v2j9I%2BlPL9doAspb%2B3vDZL%2FpHOrw07MIRm1UVQiEFjotEXt5hIHmkq33iL0pNzLxtTxRbUuy7lZ3gXBRSTFzCabZgpq5JXOehDPUVMaopMslKZL5qWxXfwTndrAUo%2BSf2ruFUY0O91JFEzcPco0hJHYDTBgfQDwB43Ee2iHf8v1MyPuyDs1npp9%2FnWFhHaZnYRb4XHXJAgTHcSI44ytJKRDiNertr5GQyQfS7wQ5MRIY3UJSRnegxDzZGhgXhYWqFzpxchqg%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:23:08,640 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1252'}
2024-03-06 17:23:08,640 ERROR   Thread-21 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:23:09,144 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1640] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/shahwaiz/UNet-DAS/1wq3lei6/output.log?Expires=1709803093&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=s%2B1u7iQWOAsWrSGO%2BRk8a4tWBlF8KWwvmojY9LHRmn1VhGLaxxFu06JQWvvbbVYDxGdhmnndCPYIgo98HpbZa3P591S%2BntRG%2FheOLkdEYvqBjb7HELC73PoSP0jemLOsdMTRrjc4Fy9xYsHAA1kdc69A9g39toKhFLHf%2BqkOICoJgsNc9fLsNsMRgUSymc4vSghOdxH4SrxnSfezW7ZeEykqZvdm8W6%2F2TeNFnlwkxDy473vn3kmHKQ%2FI%2BkQTkQU1lpZGX6bZKrixczzzk4ClFIN27T67WGvKDz%2BfMvnmTq7WdDAKCEwrsim%2BRqiF%2BXsJYeyZ11QRVnEYDtJ7hUkKA%3D%3D: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2024-03-06 17:23:09,144 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1642] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '2396'}
2024-03-06 17:23:09,145 ERROR   Thread-20 :71408 [internal_api.py:upload_file():1644] upload_file response body: 
2024-03-06 17:24:06,613 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:24:09,252 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:24:14,004 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:24:23,809 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:24:42,035 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:25:17,161 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-03-06 17:26:30,421 WARNING FileStreamThread:71408 [file_stream.py:request_with_retry():593] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/shahwaiz/UNet-DAS/1wq3lei6/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))). func: <bound method Session.post of <requests.sessions.Session object at 0x000001527FB4C460>>, args: ('https://api.wandb.ai/files/shahwaiz/UNet-DAS/1wq3lei6/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
